### PR TITLE
Feature/conditional show perf prerelease label fix

### PR DESCRIPTION
**Pull request by andywilkinshmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3398

### Change description ###
Update to latest ccd-ui-toolkit

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: test/ngIntegration/mockData/ccdCaseMock.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                        complexl2: { dl: { value: listItems[1], list_items: listItems }, dl2: { value: listItems[2], list_items: listItems } }
+                    }
+                }
+            ]
+        })
+        .setFieldProps({ display_context_parameter: "#COLLECTION(allowInsert,allowDelete)" })
+
+
+
+        // .addCaseField({ id: "coll", type: "Collection", label: "Coll wth DL", collection_field_type:
```
Comment (Daniel-Lam) [inline test/ngIntegration/mockData/ccdCaseMock.js:99]: Some commented-out code, here... maybe worth removing if no longer needed?

File: test/nodeMock/app.js [hunk 2]
```diff
@@ -2,15 +2,22 @@
...
+const CCDCaseDetails = require('./ccd/ccdCaseConfig/caseDetailsConfigGenerator');
+
+const { getDLCaseConfig} = require('../ngIntegration/mockData/ccdCaseMock');
+
 const port = 3001;
 
 
 class MockApp{
     init(){
+        this.intercepts =[];
```
Comment (Daniel-Lam) [inline test/nodeMock/app.js:20]: Being _really_ pedantic here, but spacing... recommend `this.intercepts = []`

File: test/nodeMock/app.js [hunk 3]
```diff
@@ -84,49 +100,36 @@ class MockApp{
...
 
+
 const mockInstance = new MockApp();
 module.exports = mockInstance;
 
+const args = minimist(process.argv)
+if (args.standalone){
+    mockInstance.init();
+    setUpcaseConfig();
+    // getDLCaseConfig();
```
Comment (Daniel-Lam) [inline test/nodeMock/app.js:111]: Commented-out code... candidate for removal?

**Review by Mo-Lala84 — APPROVED** [review]
approved

File: test/nodeMock/ccd/ccdApi.js [hunk 4]
```diff
@@ -84,9 +87,171 @@ class CCDApi{
...
+        const ccdWorkBasketInputGenerator = new CCDWorkBasketInputGenerator();
+        return ccdWorkBasketInputGenerator
+            .addField({ id: "simpletext", type: "Text", label: "Simple text input" })
+            .addField({ id: "radioInput", type: "FixedRadioList", label: "Simple Radio input", list: [{ code: "a", label: "A" }, { code: "b", label: "B" }, { code: "c", label: "C" }] })
+            .addField({ id: "radioYesorNo", type: "YesOrNo", label: "Simple Yes or No input" })
+            .addField({ id: "fixedListItem", type: "FixedList", label: "fixed listinput", list: [{ code: "a", label: "A" }, { code: "b", label: "B" }, { code: "c", label: "C" }] })
+            .addField({ id: "multiSelectItem", type: "MultiSelectList", label: "Multi select input", list: [{ code: "a", label: "A" }, { code: "b", label: "B" }, { code: "c", label: "C" }] })
+
+            .getConfig();
+
```
Comment (Daniel-Lam) [inline test/nodeMock/ccd/ccdApi.js:135]: Lots of whitespace throughout this file... can some of this be removed?

File: test/nodeMock/ccd/ccdCaseConfig/CCDCaseField.js [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+            this.setObjectProps(template, fieldConfig.props);
+        }
+        return template;
+    }
+
+    ConfigureCCDField(parentField, fieldConfig) {
+
+        if (fieldConfig.type === "Complex") {
+            fieldConfig.complex_fields.forEach((complexFieldConfig) => {
+                // console.log(" complex field config " + JSON.stringify(complexFieldConfig));
```
Comment (Daniel-Lam) [inline test/nodeMock/ccd/ccdCaseConfig/CCDCaseField.js:128]: Commented-out code needing removal; also, on some of the following lines

File: test/nodeMock/ccd/ccdCaseConfig/caseCreateConfigGenerator.js [hunk 6]
```diff
@@ -83,94 +55,91 @@ class CCDCaseConfig {
...
+        return this; 
+    }
+
+    setFieldTypeProps(fieldTypeprops){
+        this.checkPageAndFiedlSet();
+        this.setObjectProps(this.currentCaseField.field_type, fieldTypeprops);
+        return this; 
+    }
+
+    checkPageAndFiedlSet(){
```
Comment (Daniel-Lam) [inline test/nodeMock/ccd/ccdCaseConfig/caseCreateConfigGenerator.js:128]: Looks like a misspelling here - shouldn't it be `checkPageAndFieldSet()`?

File: test/nodeMock/ccd/ccdCaseConfig/caseDetailsConfigGenerator.js [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+
+    addHistoryTab(tabLabel){
+        const historyTab = this.addTab(tabLabel ? tabLabel : "Mock tab History");
+        const historyViewer = this.caseConfig.getCCDFieldTemplateCopy({id: "CaseHistoryViewer", type: "caseHistory", label: "Mock History View"});
+        historyViewer.value = this.caseDetailsTemplate.events;
+        historyTab.fields.push(historyViewer);
+        return this;
+    }
+
+    // addFieldToTab(tab, caseField){
```
Comment (Daniel-Lam) [inline test/nodeMock/ccd/ccdCaseConfig/caseDetailsConfigGenerator.js:73]: Commented-out code to be removed, if not needed

File: test/nodeMock/ccd/ccdCaseConfig/caseDetailsConfigGenerator.js [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+    }
+
+    addMetadata(ccdField) {
+        ccdField.metadata = true;
+        this.caseDetailsTemplate.metadataFields.push(ccdField);
+        return this;
+    }
+
+}
+
```
Comment (Daniel-Lam) [inline test/nodeMock/ccd/ccdCaseConfig/caseDetailsConfigGenerator.js:138]: Could probably lose the whitespace, here

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@andywilkinshmcts - just a couple of minor things for you to look at

File: test/nodeMock/app.js (hunk 2 above)
Comment (andywilkinshmcts) [reply test/nodeMock/app.js:20]: We need to make these changes, but not it this build - all changes from previous branch

**Review by Daniel-Lam — APPROVED** [review]
@andywilkinshmcts - approved on the basis that the review points raised will be deferred to another PR or release

**Pull request merged**